tests/test_simplates.py

```diff
@@ -38,14 +38,12 @@ def test_cant_implicitly_override_state(harness):
 
 
 def test_can_explicitly_override_state(harness):
-    output = harness.simple("[---]\n"
-        "class Output: body=media_type=charset=code=299\n"
-        "state['output'] = Output()\n"
-        "[---]\n"
-        "bar"
```

Comment:
I've added back the possibility to replace the `output` object, and more: `body` can now be overwritten, and when it is then the third page isn't rendered at all.